cli/tools/coverage.rs

```diff
@@ -221,8 +231,34 @@ impl PrettyCoverageReporter {
         println!("{}", colors::red(&line_coverage));
       }
 
+      let output_lines =
+        if let Some(original_source) = maybe_original_source.as_ref() {
+          original_source.split("\n").collect::<Vec<_>>()
+        } else {
+          lines
+        };
+
+      let output_indices = if let Some(source_map) = maybe_source_map {
+        let mut indices = uncovered_lines
+          .iter()
+          .map(|i| {
+            source_map
+              .lookup_token(*i as u32, 16)
```

Comment:
1:1 mapping also isn't going to work; should get all the source lines that the executed line represents (happens to be 1:1 in most of our generated code but brittle).